duckduckgo/Android
# Implement basic web browser

Reviewer: @subsymbolic 
Asana: https://app.asana.com/0/0/341403501734105

**Description**: Remove all the old code and implement a basic web browser with a minSdk 16, latest build tools and support libraries, MVP implementation of the presentation layer to be able to test the business logic and zero lint error and warnings


**Steps to test this PR**:
1. travis will build and run all the test and should fail if there are tests failing or any lint error or warning


###### Reviewer Checklist:


###### PR DRI Checklist:
